cats_and_dogs.md

```diff
@@ -0,0 +1,126 @@
+![https://xkcd.com/1558/](https://imgs.xkcd.com/comics/vet.png)
+
+# Find me pets
```

Comment:
Should this title perhaps be "Cats and dogs" to match the problem name?

```suggestion
# Cats and dogs
```

Reply:
For style points, suggest moving the heading above the image to (consistent with the formatting for the other problems)